bin/src/run/index.ts

```diff
@@ -126,6 +126,10 @@ function execute(configFile: string, configs: InputOptions[], command: any) {
 			if (optionError) inputOptions.onwarn({ code: 'UNKNOWN_OPTION', message: optionError });
 
 			return build(inputOptions, outputOptions, warnings, command.silent);
+		}).then(results => {
+			if (!results.every(Boolean)) {
+				process.exit(1);
```

Comment:
This still seems useful?

Reply:
Ah, it seems like this was just needed because errors were no longer propagated.

Reply:
I don't think so. It's error batching.